tests/FormatterTest.php

```
@@ -5,9 +5,28 @@
+		foreach (
+			[
+				'custom_formatters'          => [],
+				'format_options'             => [],
+				'single_value_formatters'    => [],
+				'builtin_formats_registered' => false,
+			] as $property => $value
+		) {
+			$reflection = new \ReflectionProperty( Formatter::class, $property );
+			if ( PHP_VERSION_ID < 80100 ) {
+				// @phpstan-ignore method.deprecated
+				$reflection->setAccessible( true );
```

PHPStan (2.2.7)

[error] No error with identifier method.deprecated is reported on line 26.


(ignore.unmatchedIdentifier)